renku/cli/dataset.py

```diff
@@ -81,18 +81,25 @@
 import click
 from click import BadParameter
 
+from renku.models._tabulate import tabulate
 from renku.models.datasets import Author
 
 from ._client import pass_local_client
 from ._echo import progressbar
 
 
-@click.group()
+@click.group(invoke_without_command=True)
 @click.option('--datadir', default='data', type=click.Path(dir_okay=True))
+@pass_local_client(clean=True, commit=True)
 @click.pass_context
-def dataset(ctx, datadir):
+def dataset(ctx, client, datadir):
     """Handle datasets."""
     ctx.meta['renku.datasets.datadir'] = datadir
+    table = tabulate(
+        client.datasets.values(),
+        headers=["short_id", "name", "authors_csv", "created"],
```

Comment:
I'm coming here a bit late, but couldn't `authors_csv` be just `authors`?

Reply:
Yes, it can, but then the output is kinda ugly.

Example:
```SHORT_ID    NAME         AUTHORS                                                             CREATED
----------  -----------  ------------------------------------------------------------------  -------------------
87c46dd9    somedataset  [Author(name='sam', email='contact@justsam.io', affiliation=None)]  2019-02-18 13:47:10
5aa65a26    dataset1234  [Author(name='sam', email='contact@justsam.io', affiliation=None)]  2019-02-18 13:47:39
```

That's why I went for authors_csv property where I merge the names of all authors into CSV format.

Reply:
I just meant the name of the column. The `csv` part isn't needed imo

Reply:
Yea, I understand. I guess the same argument could be also applied for `short_id` header. Should we  change our internal wrapper of tabulate to achieve this renaming or we have already this functionality somewhere?

Reply:
We could define headers as mapping and use keys for getting the attributes and values as column names.

```python
def tabulate(..., headers, ...):
    ...
    if isinstance(headers, dict):
        attrs, names = headers.keys(), headers.values()
    else:
        attrs = names = headers
    table = [(
        format_cell(cell, datetime_fmt=datetime_fmt)
        for cell in attrgetter(*attrs)(c)
    ) for c in collection]
    return tblte(table, headers=[h.upper() for h in names], **kwargs)
```